scripts/tracecontext-integration-test.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+set -e
+# hard-coding the git tag to ensure stable builds.
+TRACECONTEXT_GIT_TAG="98f210efd89c63593dce90e2bae0a1bdcb986f51"
+# clone w3c tracecontext tests
+mkdir -p target
+rm -rf ./target/trace-context
+git clone https://github.com/w3c/trace-context ./target/trace-context
+cd ./target/trace-context && git checkout $TRACECONTEXT_GIT_TAG && cd -
+pip3 install setuptools;
+pip3 install aiohttp; 
+(cd validation_server; npm i)
```

Comment:
This is the workaround for https://github.com/open-telemetry/opentelemetry-js/pull/1380/files#r464520833, but it's not a real workaround because it's not testing against the latest commit but the latest released version I think